### PR TITLE
sort -C and -u flags

### PR DESCRIPTION
**Pull request by binjip978**

### CODE REVIEW

File: cmds/core/sort/sort_test.go [hunk 1]
```diff
@@ -1,39 +1,94 @@
...
+func TestSortStdin(t *testing.T) {
+	for _, tt := range []struct {
+		name    string
+		params  params
+		input   string
+		want    string
+		wantErr error
+	}{
+		{
+			name:    "unique no dublicates",
```
Comment (rminnich) [inline cmds/core/sort/sort_test.go:27]: dublicates -> duplicates

**Review by rminnich — COMMENTED** [review]
substantial improvement! I note the one spelling change.

File: cmds/core/sort/sort.go [hunk 2]
```diff
@@ -65,45 +97,84 @@ func readInput(w io.Writer, f *os.File, args ...string) error {
...
+	_, err := to.Write([]byte(s))
+	return err
 }
 
 func main() {
 	flag.Parse()
-	if err := readInput(os.Stdout, os.Stdin, flag.Args()...); err != nil {
+	p := params{reverse: *reverse, ordered: *ordered, outputFile: *outputFile, unique: *unique}
+	if err := command(os.Stdin, os.Stdout, os.Stderr, p, flag.Args()).run(); err != nil {
+		if err == errNotOrdered {
```
Comment (rminnich) [inline cmds/core/sort/sort.go:175]: should you use errors. here? 

Also, does log.Fatal make sense instead of os.Exit? I don't know, so I'm asking :-)

Comment (binjip978) [reply cmds/core/sort/sort.go:175]: log.Fatal will print additional stuff that we don't need there. From POSIX spec: `Same as -c, except that a warning message shall not be sent to standard error if disorder or, with -u, a duplicate key is detected.`

errors.Is will work, but I think we explicitly search for specific error here, so `==` should be ok.

**Pull request merged**